### DISCUSSION TITLE
The context name of <FluentEditForm> if using <AuthorizeView>, Context name conflict ?

### DISCUSSION THREAD

**Discussion by andrewwang1tw** [microsoft/fluentui-blazor#2457] [Q&A] [answered]
Below code, uses "FluentEditForm" in "AuthorizeView"
```
<AuthorizeView>
    <Authorized>
        <FluentEditForm  Model="productItemModel"
```

Here is the compile error
```
RZ9999 The child content element 'ChildContent' of component 'FluentEditForm' 
uses the same parameter name ('context') as enclosing child content element 'Authorized' of component 
'AuthorizeView'. 

Specify the parameter name like: '<ChildContent Context="another_name"> to resolve the ambiguity
```

If setting the context name of "FluentEditForm" explicitly, **Context="FluentEditForm"**, then it works fine.
```
<AuthorizeView>
    <Authorized>
        <FluentEditForm Context="FluentEditForm" Model="productItemModel"
``` 

But from the [https://www.fluentui-blazor.net/](url), just cannot find the related documentation about "FluentEditForm".

**Comment by vnbaaij** [comment, marked as answer]
We have indeed not documented `FluentEditForm` yet. It doesn't offer anything extra besides when you are using a `FluentWizard`. Anyways, this is a standard Blazor error/message and has nothing to do with the `FluentEditForm`. Setting the contect name is exactly what needs to be done.

**Reply by andrewwang1tw** [reply]
OK, Thanks!
